x509-cert/src/lib.rs

```diff
@@ -20,7 +20,7 @@ extern crate alloc;
 extern crate std;
 
 #[macro_use]
-mod macros;
+pub mod macros;
```

Comment:
It's not necessary (or desirable) to make this `pub`

```suggestion
mod macros;
```

Reply:
Changed.